### PR TITLE
[Testmerge] The Harbor update

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
Credits:
[Acantharctia](https://github.com/Acantharctia) - Map changes, the original idea of the harbor, job descriptions
Cyan - Ideas, design
Wehstone - Axian lore
Myself - Code, ideas, design, lesser map changes

64MlIBBFwu
--TBA--

Map:
Adds a Harbor district, which consists of a guardhouse (for harbormaster and longshoreman), a watch tower, townie houses and rentable shops, along with the dock - which has a couple small ships and one big one, from which the crew will unload coffers and deliver them to the merchant

New roles
Harbormaster (1 slot, Axian, Middle-Age+, Abyssor): Ensure safety on the harbor, make sure the merchant is paying your crew enough.
Longshoreman (4 slots, Axian, Abyssor): Haul cargo to the merchant, help with ensuring safety

New advclasses:
Pirate (Adventurer, Axian, Matthios/Abyssor): yarr, with cool pirate hats
Smuggler (Pilgrim, Axian, Matthios/Abyssor): yarr, but with illegal goods

New structures:
Rollers: (Carpentry: 2 log, 1 chain)
Flat boards with wheels and chains you can link together and pull them, crates and barrels on top of rollers will be pulled along

New features:
Axian Lore - species description, accent, random names, aswell as the roles mentioned previously
GOLDFACE can now purchase things through shipments, generally cheaper, while the instant teleport buy is slightly more expensive
Steward now has a new ring of keys, consisting of keys for rentable harbor shops

Bugfixes:
GOLDFACE no longer randomizes costs +-14%

### CODE REVIEW

**Comment by Bapldap** [comment]
The race restriction in general seems unfair. I doubt that there will be enough axian players to fill the slots on the roles every round.
Also, not sure if my current PR that expands the gold-face to have a wardrobe license will affect this.

**Comment by Stagrawl** [comment]
Big cats can swim.

**Comment by K4rlox** [comment]
hehe
shonk 
image

**Comment by wow50000** [comment]
not a fan of race locking these ?

**Comment by Mogurk** [comment]
orcs too! if possible

**Comment by DorfGuy1** [comment]
Just make sure it's all the 'outsider' races, that can do it. The sketchy. So no elves, but maybe dark elves. Half orcs are mutts rejected by society, so obviously.

File: code/modules/jobs/job_types/roguetown/garrison/harbormaster.dm [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 5, TRUE) 
+		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/swimming, 6, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 2, TRUE) 
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/labor/fishing, 2, TRUE)
+		H.change_stat("strength", 5)
```
[the remark this answers is not in the document]
Comment (CVSthePharmacy) [reply code/modules/jobs/job_types/roguetown/garrison/harbormaster.dm:56]: It's balanced

**Comment by CVSthePharmacy** [comment]
snowflakey as it's a town role with a specific race, and not human by default.

**Pull request closed without merging**